src/cli.ts

```diff
@@ -22,6 +22,7 @@ prog.command('dev')
 	.option('-p, --port', 'Specify a port')
 	.option('-o, --open', 'Open a browser window')
 	.option('--dev-port', 'Specify a port for development server')
+	.option('--dev-port-timeout', 'Specify a timeout for the dev port watcher', 5000)
```

Comment:
Added the duplicate default value to communicate the default value via CLI help menu.